### DISCUSSION TITLE
Duration time now shown for MP3 HLS

### DISCUSSION THREAD

**Discussion by feather42** [aclap-dev/video-downloadhelper#190] [Help and Q&A]
### Your question

Hi,
After the latest update when I press the VDH button I get the items I can download listed but they no longer have times shown.
Take an example I go to a site to download a file the default file might be 10mins long I use the option in the site to set to 15mins long normally
![Screenshot from 2024-02-20 15-54-05](https://github.com/aclap-dev/video-downloadhelper/assets/127102751/cb6a599e-13b7-4e58-ad09-34ad6a103257)
 VDH will show many hits and among them will be a 10 mins & a 15mins file so I can easily see which I want to download.
Frequently there are many hits shown on websites and I use the time to work out which the hit is that I want to DL.
See attached screenshot from a webpage with a single audio download on it I can understand which hit I need to download, normally the time would guide me.

Am I missing a new setting or something have the times been taken away?

Thanks for any help you can give.

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "/home/newhamcm/vdhcoapp-2.0.10/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/newhamcm",
  "converterBinary": "/home/newhamcm/vdhcoapp-2.0.10/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
That's a regression. Can you share the link to that website?

**Reply by feather42** [reply]
Its a page on headspace.com
https://my.headspace.com/player/192?contentId=1510&contentType=Meditation&edhs=true&mode=meditate&trackingName=Meditation
